inference/generativeai/llm-workshop/lab2-stable-diffusion/option2-aitemplate/sd_txt2img.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<p>Notebook Title should always be Heading 1</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>